recipes/ohnet/config.yml

```diff
@@ -0,0 +1,5 @@
+versions:
+  "1.36.5113":
+    folder: all
+  "1.36.5182":
```

Comment:
This newer version is missing in conandata.yml. But, please, let's only the latest version in case you don't need both now.